bundle-ocp/manifests/das-scheduler-config_v1_configmap.yaml

```diff
@@ -22,3 +18,6 @@ data:
       - name: MigAccelerator
         args:
           namespace: das-operator
+kind: ConfigMap
+metadata:
+  name: das-scheduler-config
```

Comment:
we also have this file in the `deploy` folder?

Reply:
I wasn't able to find it in the deploy folder